app/scripts/directives/seekBar.js

```diff
@@ -30,6 +30,10 @@
                     return {width: percentString()};
                 };
                 
+                scope.thumbStyle = function(){
+                    return {width: 10};
```

Comment:
so, i'm playing with it and used ```left: 10```. I don't know how to make the thumb nail move with the percentage bar